classes/ColdTrick/FriendRequest/PageMenu.php

```diff
@@ -14,7 +17,9 @@ class PageMenu {
 	 *
 	 * @return void|\ElggMenuItem[]
 	 */
-	public static function registerCleanup($hook, $type, $return_value, $params) {
+	public static function registerCleanup(Hook $hook) {
+		
+		$return_value = $hook->getValue();
 		
 		if (empty($return_value) || !is_array($return_value)) {
```

Comment:
this will fail in 3.0 as `$return_value` now is an instance of `MenuItems` so we just remove this check

Reply:
What's a reason to unset `friends:of` menu item?

Reply:
since with friend request both user A and B are friends of each other (so A => B and B => A) there is no difference between the two listings. So why show them?

Reply:
Ok. We just use the 'followers/following' case on some sites, so I asked about it.